Logic/Config.cs

```diff
@@ -17,4 +19,14 @@ public class Config
 
         return null;
     }
+
+    public void Deserialize(string configurationFilePath)
+    {
+        using var configurationFile = File.Open(configurationFilePath, FileMode.Open, FileAccess.Write);
+        JsonSerializer.Serialize(
```

Comment:
Method is called `Deserialize`, but you are serializing - where is a mistake?

Reply:
Resolved in the recent commit